.changeset/early-colts-sneeze.md

```diff
@@ -0,0 +1,5 @@
+---
+"@navikt/ds-react": patch
+---
+
+Autocomplete in combobox will now change formatting of the typed word to match upper-/lowercase of word being autocompleted
```

Comment:
Change this to say we DO NOT change formatting during typing, but use casing from the autocompleted word when selecting/pressing enter.